rosbag2/include/rosbag2/sequential_reader.hpp

```diff
@@ -100,10 +101,8 @@ class ROSBAG2_PUBLIC SequentialReader
   virtual std::vector<TopicMetadata> get_all_topics_and_types();
 
 private:
-  std::unique_ptr<rosbag2_storage::StorageFactoryInterface> storage_factory_;
-  std::shared_ptr<SerializationFormatConverterFactoryInterface> converter_factory_;
-  std::shared_ptr<rosbag2_storage::storage_interfaces::ReadOnlyInterface> storage_;
-  std::unique_ptr<Converter> converter_;
+  class ReaderImpl;
+  std::unique_ptr<ReaderImpl> reader_pImpl_;
```

Comment:
The names of variables (including function parameters) and data members are all lowercase, with underscores between words. Data members of classes (but not structs) additionally have trailing underscores. For instance: a_local_variable, a_struct_data_member, a_class_data_member_.

https://google.github.io/styleguide/cppguide.html#Variable_Names